internal/fakecgo/trampolines_stubs.s

```diff
@@ -31,7 +33,7 @@ TEXT _pthread_create(SB), NOSPLIT, $0-0
 	JMP purego_pthread_create(SB)
 	RET
 
-TEXT _pthread_detach(SB), NOSPLIT, $0-12
+TEXT _pthread_detach(SB), NOSPLIT, $0-0
```

Comment:
Is this change fine?

Reply:
Yes. It was a lay over from #47 but should have been replaced with `0-0`. It's because the 12 means 12 bytes of arguments but these trampolines don't need to make any room for the arguments since call5 already does that.